modules/ROOT/pages/update-workspace.adoc

```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
```

Comment:
HIDDEN TEXT EDIT

= Updating Workspaces
ifndef::env-site,env-github[]

Reply:
I would like to try something new.
I know that our team uses imperative voice (in this case, Update) for tasks, but this is not a task.
However, the gerund case (in this case, Updating) sounds a bit like a task, too.
What would you think of "Workspace Updates:?